lldb/test/API/functionalities/data-formatter/data-formatter-stl/libcxx-simulators/string/main.cpp

```diff
@@ -183,31 +183,40 @@ template <class _CharT, class _Traits, class _Allocator> class basic_string {
     };
   };
 
+#if COMPRESSED_PAIR_REV == 0
   std::__lldb::__compressed_pair<__rep, allocator_type> __r_;
+#define __R_ __r_
```

Comment:
Probably more readable if I just replaced all the `__r_.first().XXX` accesses with calls to `setXXX` or something